src/main/resources/native/android/dalvik/source/MainActivity.java

```diff
@@ -305,6 +306,27 @@ private void notifyLifecycleEvent(String event) {
         }
     }
 
+    private static IntentHandler intentHandler;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5148382)